orchestrator/orchestrator.go

```diff
@@ -62,42 +64,43 @@ func (o *Orchestrator) Start(ctx context.Context) error {
 			return fmt.Errorf("anvil instance %s failed to start: %w", anvil.Name(), err)
 		}
 	}
-
-	if err := o.WaitUntilAnvilsAreReady(); err != nil {
-		return fmt.Errorf("anvil instances failed to get ready: %w", err)
-	}
-
-	for _, opSim := range o.L2OpSims {
+	for _, opSim := range o.l2OpSims {
 		if err := opSim.Start(ctx); err != nil {
 			return fmt.Errorf("op simulator instance %s failed to start: %w", opSim.Name(), err)
 		}
 	}
 
+	if err := o.WaitUntilAnvilsAreReady(); err != nil {
```

Comment:
this has to go before the opSim instances are started, otherwise the clocks will drift since the opsim startup triggers transactions on l2 that will mine blocks and start the clock.